SYCL/Basic/parallel_for_range_roundup.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
-// RUN: env SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE=1 %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_PLACEHOLDER env SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE=1 %t.out %GPU_CHECK_PLACEHOLDER
```

Comment:
My guess is that one can imply `xxx_RUN_PLACE` contains `env` (or other OS-dependent stuff) already. Should `env` be present on the `RUN` command in such a case, then?